build/configs/artik05x/artik05x_download.sh

```diff
@@ -36,6 +36,7 @@ ARTIK05X_DIR_PATH=${CONFIGS_DIR_PATH}/artik05x
 SCRIPTS_PATH=${ARTIK05X_DIR_PATH}/scripts
 
 TIZENRT_BIN=${OUTPUT_BINARY_PATH}/tinyara_head.bin
```

Comment:
I think we should change name of tinyara_head.bin to tinyara_kernel.bin for better clarity on what the binary is for.
and TIZENRT_BIN to TIZENRT_KERNEL_BIN.

Reply:
I agree with your suggestion. But in that case, we must have the binary name as tinyara_head.bin in normal case and tinyara_kernel.bin in protected case. This requires changes in many files and also makes things a bit complex. So, I think we must keep the current status.